Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM library/node:6.3
+
+WORKDIR /src
+
+COPY package.json .
```

Comment:
Yes, if we care about caching we'll have to change it a bit. Also as I've mentioned currently npm will automatically run the postinstall script which will not work as expected. That script was added for heroku, and can be removed once we have properly working dockerfile and deployment procedure.